GaussianPathState start
PathState goal
--- 
string out